### PR TITLE
Fix the issues with the travis encryption from forks

### PR DESCRIPTION
**Pull request by markglh**
Travis disables decryption when the pull request is from a fork, this causes the build to fail. We don't need to decrypt the keys unless it's a master build, so we can simply disable this form PRs.

### CODE REVIEW

**Comment by agaro1121** [comment]
@markglh Guess this is ok for now?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# TEST Fork
+
```
Comment (markglh) [inline README.md]: Just remove this @agaro1121

**Comment by agaro1121** [comment]
@markglh Done

**Comment by markglh** [comment]
@etspaceman - release will be with you shortly: https://github.com/WW-Digital/reactive-kinesis/releases/tag/v0.5.6

**Pull request merged**